src/main/java/org/jboss/xavier/analytics/functions/HelperFunctions.java

```diff
@@ -85,7 +85,8 @@ boolean isConvertible()
     public enum FlagUnsuitabilityForOSPTarget{
         RDM_DISK(WorkloadInventoryReportModel.RDM_DISK_FLAG_NAME, true),
         TOO_MANY_NICS(WorkloadInventoryReportModel.MORE_THAN_4_NICS_FLAG_NAME, true),
-        SHARED_DISK(WorkloadInventoryReportModel.SHARED_DISK_FLAG_NAME,true);
+        SHARED_DISK(WorkloadInventoryReportModel.SHARED_DISK_FLAG_NAME,true),
+        NUMA_PINNINGS(WorkloadInventoryReportModel.NUMA_PINNINGS_FLAG_NAME, false);
```

Comment:
Following https://issues.redhat.com/browse/MIGENG-358?focusedCommentId=14000961&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-14000961, please change this to become `CPU_AFFINITY`